### PR TITLE
Flutter v2 gen2 comment fixes

### PR DESCRIPTION
**Pull request by khatruong2009**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx [hunk 1]
```diff
@@ -0,0 +1,200 @@
+<BlockSwitcher>
+<Block name="Amplify CLI (Create)">
+
+> Prerequisites: [Install and configure](/gen1/prev/[platform]/start/project-setup/prerequisites/#install-and-configure-the-amplify-cli) the Amplify CLI in addition to the Amplify libraries and [necessary dependencies](/gen1/prev/[platform]/build-a-backend/auth/set-up-auth/#install-amplify-libraries).
+
+To start provisioning auth resources in the backend, go to your project directory and **execute the command**:
+
+```bash
```
Comment (josefaidt) [inline src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx:8]: can we add `title="Terminal" showLineNumbers={false}` to each of these bash snippets where we're showing a command that should be executed in the terminal?

File: src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+<Fragments fragments={{ android: android4 }} />
+
+import flutter5 from '/src/fragments/lib/auth/flutter/getting_started/12_amplifyConfig.mdx';
+
+<Fragments fragments={{ flutter: flutter5 }} />
+
+</Block>
+
+<Block name="Amplify CLI (Import)">
```
[the remark this answers is not in the document]
Comment (khatruong2009) [reply src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx:45]: It was like that in the `lib` pages, I just copied them over to `lib-v1` since there were imports in `lib-v1` that were references these same pages but in `lib`

File: src/fragments/lib-v1/auth/native_common/getting_started/common.mdx [hunk 3]
```diff
@@ -36,15 +36,15 @@ import flutter8 from '/src/fragments/lib-v1/auth/flutter/getting_started/20_inst
 
 The most common way to use Authentication with Amplify is via the Amplify CLI, which allows you to create new Amazon Cognito resources or import existing ones. However, you can also use the Amplify Studio console to configure authentication or use the `Amplify.configure()` method to set up authentication with existing resources.
 
-import ios50 from '/src/fragments/lib/auth/ios/getting_started/50_configureBackend.mdx';
+import ios50 from '/src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx';
```
Comment (Jordan-Nelson) [inline src/fragments/lib-v1/auth/native_common/getting_started/common.mdx:39]: These 3 fixes from lib to lib-v1 seem to already exist in main. Lets only fix the ones that we introduced in the PR. This file doesn't look like it was touched in the current PR until now.

File: src/fragments/lib-v1/storage/native_common/getting-started/common.mdx [hunk 4]
```diff
@@ -126,8 +127,4 @@ Congratulations! You've uploaded a file to an s3 bucket. Check out the following
 - [Remove Files](/gen1/[platform]/prev/build-a-backend/storage/remove/)
 - [File Access Levels](/gen1/[platform]/prev/build-a-backend/storage/configure-access/)
 - [Using Lambda Triggers](/gen1/[platform]/prev/build-a-backend/storage/lambda-triggers/)
-</InlineFilter>
-
-<InlineFilter filters={['android', 'swift']}>
```
Comment (Jordan-Nelson) [inline src/fragments/lib-v1/storage/native_common/getting-started/common.mdx:131]: Why is this removed?

File: src/pages/gen1/[platform]/prev/build-a-backend/graphqlapi/mutate-data/index.mdx [hunk 5]
```diff
@@ -56,10 +56,6 @@ import flutter3 from '/src/fragments/lib-v1/graphqlapi/flutter/mutate-data.mdx';
 
 In this guide, you will learn how to create, update, and delete your data using Amplify Libraries' GraphQL client.
 
-Before you begin, you will need:
-
-- An [application connected to the API](/gen1/[platform]/prev/build-a-backend/graphqlapi/set-up-graphql-api/)
```
Comment (Jordan-Nelson) [inline src/pages/gen1/[platform]/prev/build-a-backend/graphqlapi/mutate-data/index.mdx:61]: I think we discussed removing this, but I don't remember why. Maybe we though it was a vCurrent reference from vPrev? It looks fine to me since it is to /prev from /prev. Let's leave this if there wasn't another reason to remove it.

Comment (Jordan-Nelson) [reply src/pages/gen1/[platform]/prev/build-a-backend/graphqlapi/mutate-data/index.mdx:61]: Same for the next 2 files

**Review by Jordan-Nelson — APPROVED** [review]
